domain-models-runtime/src/main/resources/templates/db_scripts/metadata.ftl

```diff
@@ -22,33 +22,36 @@ CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.set_${table.tableName}_md
 RETURNS TRIGGER AS $$
 DECLARE
   createdDate timestamp WITH TIME ZONE;
-  createdBy text ;
+  createdBy text;
   updatedDate timestamp WITH TIME ZONE;
-  updatedBy text ;
-  injectedId text;
+  updatedBy text;
+
+  dateFormat text := 'YYYY-MM-DD"T"HH24:MI:SS.MSOF';
+  injectedMetadata jsonb;
+
 BEGIN
+  SET TIME ZONE 'UTC';
+
   createdBy = NEW.created_by;
   createdDate = NEW.creation_date;
   updatedDate = NEW.jsonb->'metadata'->>'updatedDate';
   updatedBy = NEW.jsonb->'metadata'->>'updatedByUserId';
 
-  if createdBy ISNULL then
-    createdBy = 'undefined';
-  end if;
-  if updatedBy ISNULL then
-    updatedBy = 'undefined';
-  end if;
   if createdDate IS NOT NULL then
```

Comment:
Can `createdDate` been removed? If is used two times only where we can directly write `NEW.creation_date`